components/renku_data_services/notifications/orm.py

```diff
@@ -0,0 +1,62 @@
+"""SQLAlchemy schemas for the notifications database."""
+
+from datetime import datetime
+from typing import Optional
+
+from sqlalchemy import DateTime, MetaData, String, func
+from sqlalchemy.orm import DeclarativeBase, Mapped, MappedAsDataclass, mapped_column
+from ulid import ULID
+
+from renku_data_services.notifications import models
+from renku_data_services.utils.sqlalchemy import ULIDType
+
+metadata_obj = MetaData(schema="notifications")
+
+
+class BaseORM(MappedAsDataclass, DeclarativeBase):
+    """Base class for all ORM classes."""
+
+    metadata = metadata_obj
+
+
+class AlertORM(BaseORM):
+    """The alerts."""
+
+    __tablename__ = "alerts"
+
+    id: Mapped[ULID] = mapped_column("id", ULIDType, primary_key=True, default=lambda: str(ULID()), init=False)
+    """ID of the alert."""
+
+    title: Mapped[str] = mapped_column("title", String(), nullable=False)
+    """Title of the alert."""
+
+    message: Mapped[str] = mapped_column("message", String(), nullable=False)
+    """Message of the alert."""
+
+    user_id: Mapped[str] = mapped_column("user_id", String(), nullable=False)
```

Comment:
If possible we should have a foreign key constraint for `user_id`. Otherwise we can add alerts for users that do not exist.


```suggestion
    user_id: Mapped[str] = mapped_column(ForeignKey(UserORM.keycloak_id, ondelete="CASCADE"), nullable=False, index=True)
```

with

```python
from renku_data_services.users.orm import UserORM
```